manifests/nimbus-devtools.yml

```diff
@@ -0,0 +1,12 @@
+---
+description: nimbus-devtools
+repo-prefix: nimbusdevtools
+active: true
+private-repo: false
+branch: main
+additional-emails: ["brennie@mozilla.com"]
+artifacts:
+  - web-ext-artifacts/nimbus-devtools.xpi
+addon-type: privileged
```

Comment:
For my own knowledge, how do we know the values of "artifacts" and "addon-type"? (Especially given that this repo is empty right now?)

Reply:
It's by convention, `web-ext-artifacts/` is the default directory where `web-ext` build tool outputs `.xpi` archive. The name of the `.xpi` archive comes from the `manifest.json` file. The `privileged` type is for AMO or self-hosted `mozilla-extension` add-ons.